inc/api.php

```
@@ -430,51 +430,64 @@ function djz_get_gamipress_user_data($request)
-        $ach_thumb_ids = [];
-        foreach ($all_achievements as $post) {
-            $tid = get_post_thumbnail_id($post->ID);
-            if ($tid) $ach_thumb_ids[] = (int) $tid;
-        }
-        if (!empty($ach_thumb_ids)) {
-            $ach_thumb_ids = array_unique($ach_thumb_ids);
-            if (function_exists('update_meta_cache')) update_meta_cache('post', $ach_thumb_ids);
-            // Priming focado apenas no post object cache para performance máxima.
-            if (function_exists('_prime_post_caches')) _prime_post_caches($ach_thumb_ids, false, true);
+    if (false === $static_achievements) {
+        $all_achievements = get_posts([
```

PHPMD (2.15.0)

[error] The variable $all_achievements is not named in camelCase. (undefined)

(CamelCaseVariableName)